doc/reference/kernel/other/float.rst

```diff
@@ -193,6 +193,31 @@ when there is no need to do so.
 When the thread again needs to use the floating point registers it can re-tag
 itself as an FPU user or SSE user by calling :cpp:func:`k_float_enable()`.
 
+
+RISCV architecture
+------------------
+
+On the RISCV architecture, the kernel treats each thread as a non-user
+or FPU user and the thread must be tagged by one of the
+following techniques.
+
+* A statically-created RISCV thread can be tagged by passing the
+  :c:macro:`K_FP_REGS` option to :c:macro:`K_THREAD_DEFINE`.
+
+* A dynamically-created RISCV thread can be tagged by passing the
+  :c:macro:`K_FP_REGS` to :cpp:func:`k_thread_create()`.
+
+During thread context switching the RISCV kernel saves the *callee-saved*
+floating point registers, if the switched-out thread has been using them.
+Additionally, the *caller-saved* floating point registers are saved on
+the thread's stack. If the switched-in thread has been using the floating
```

Comment:
same here, should this read ``If the switched-in thread is tagged as an FPU-user, the kernel...``